cmd/jaeger/internal/integration/span_writer.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package integration
+
+import (
+	"context"
+
+	"github.com/jaegertracing/jaeger/model"
+	"github.com/jaegertracing/jaeger/storage/spanstore"
+	jaeger2otlp "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/testbed/testbed"
+)
+
+var _ spanstore.Writer = (*spanWriter)(nil)
```

Comment:
io.Closer to clean up the sender?

Reply:
Yes, added it

Reply:
Actually, the sender doesn't have the functionality to close the client, so I use OTLP exporter instead.